usage.ipfs.js

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright(c) ConsenSys Software Inc.
+ * Copyright(c) https://consensys.net/
+ * MIT Licensed
+ */
+
+/* eslint-disable */
+
+import { config as loadEnv } from 'dotenv';
+import path from 'path';
+import { SDK, Auth, TEMPLATES, Metadata } from './index.js';
+loadEnv();
+
+const projectId = process.env.INFURA_IPFS_PROJECT_ID;
+const projectSecret = process.env.INFURA_IPFS_PROJECT_SECRET;
+const ipfsUrl = process.env.INFURA_IPFS_ENDPOINT;
```

Comment:
Could we add these three to .env.template

Reply:
2 have been added / 1 removed (INFURA_IPFS_ENDPOINT)